website/orders/models.py

```diff
@@ -265,6 +265,14 @@ class Shift(models.Model):
         "possible, even in the given time frame.",
     )
 
+    finalized = models.BooleanField(
```

Comment:
We should also prevent changing `Order`s themselves when their shift has changed